rooms/default/special/remains.py

```diff
@@ -46,18 +47,16 @@ def action(user, reply, text):
 
 			user.give_gold(random.randrange(12, 72))
 
-			items.append(('loot', 'tooth'))
-			items.append(('loot', 'tooth'))
+			user.add_item('loot', 'tooth', count=2)
 
 			for it in items:
-				user.add_item(it[0], it[1])
+				user.add_item(it.group, it.name, count=it.count)
```

Comment:
What about context?

Reply:
not sure, but maybe we should include it too

Reply:
I've just saved original behavior